src/rgw/store/dbstore/sqlite/sqliteDB.cc

```diff
@@ -21,9 +21,34 @@
     ret = 0;					\
   } while(0);
 
+#define MAX_RETRIES 200 // Make it configurable
+
+#define EXECUTE_STMT(dpp, rc, stmt) \
```

Comment:
because this is c++, and because of improved compilers, you should probably just make a static inline function here

Reply:
This macro is executing arbitrary statements, and is called from other macros, so I suspect it has to be a macro for now.